src/server/utilities/cspHeader/index.js

```diff
@@ -70,81 +68,14 @@ const advertisingDirectives = {
     'https://t.effectivemeasure.net',
     'https://tags.crwdcntrl.net',
     'https://tpc.googlesyndication.com',
-    'https://gn-web-assets.api.bbc.com',
     'https://www.googletagservices.com',
     'https://securepubads.g.doubleclick.net',
     'https://bbc.gscontxt.net',
-    'https://adservice.google.co.ve',
-    'https://adservice.google.ru',
-    'https://adservice.google.co.in',
-    'https://adservice.google.com.br',
-    'https://adservice.google.com.mx',
-    'https://adservice.google.com.ec',
-    'https://adservice.google.cl',
-    'https://adservice.google.com.eg',
-    'https://adservice.google.com.sa',
-    'https://adservice.google.com.co',
-    'https://adservice.google.com.ar',
-    'https://adservice.google.com.ua',
-    'https://adservice.google.com.pe',
-    'https://adservice.google.kz',
-    'https://adservice.google.com.hk',
-    'https://adservice.google.es',
-    'https://adservice.google.co.jp',
-    'https://adservice.google.ae',
-    'https://adservice.google.com.tw',
-    'https://adservice.google.com.uy',
-    'https://adservice.google.co.il',
-    'https://adservice.google.com.pr',
-    'https://adservice.google.com.bo',
-    'https://adservice.google.iq',
-    'https://adservice.google.com.gt',
-    'https://adservice.google.com.sg',
-    'https://adservice.google.com.tr',
-    'https://adservice.google.com.pk',
-    'https://adservice.google.de',
-    'https://adservice.google.ca',
-    'https://adservice.google.com.vn',
-    'https://adservice.google.com.py',
-    'https://adservice.google.com.ng',
-    'https://adservice.google.jo',
-    'https://adservice.google.se',
-    'https://adservice.google.com.au',
-    'https://adservice.google.it',
-    'https://adservice.google.com.kh',
-    'https://adservice.google.co.nz',
-    'https://adservice.google.az',
-    'https://adservice.google.fi',
-    'https://adservice.google.com.ro',
-    'https://adservice.google.co.id',
-    'https://adservice.google.ie',
-    'https://adservice.google.cz',
-    'https://adservice.google.com.cy',
-    'https://adservice.google.at',
-    'https://adservice.google.nl',
-    'https://adservice.google.cv',
-    'https://adservice.google.com.mt',
-    'https://adservice.google.dk',
-    'https://adservice.google.com.af',
-    'https://adservice.google.fr',
-    'https://adservice.google.no',
-    'https://adservice.google.com.sv',
-    'https://adservice.google.hn',
-    'https://adservice.google.be',
-    'https://adservice.google.com.ni',
-    'https://adservice.google.ch',
-    'https://adservice.google.co.kr',
-    'https://adservice.google.pl',
-    'https://adservice.google.com.mm',
-    'https://adservice.google.so',
-    'https://adservice.google.co.tz',
-    'https://adservice.google.com.np',
-    'https://adservice.google.lk',
-    'https://adservice.google.lv',
-    'https://adservice.google.ge',
+    ...adServiceCountryDomains,
```

Comment:
A minor suggestion for naming – should we replace `ad` with `advertising`? This is might be more identifiable. We also use `advertisingDirectives` on L12.

Reply:
Thats a good shout. Will do that now.